### PR TITLE
Hono

### PR DESCRIPTION
**Pull request by nuoanunu**

### CODE REVIEW

**Comment by llamatester** [comment]
The hono.ts adapter exports: 
        
 
 _______________________________________
    Fees for 16/9/2023
    _______________________________________
    
    ETHEREUM 👇
    Backfill start time: 12/8/2023
    NO METHODOLOGY SPECIFIED
    Daily revenue: 0
    Timestamp: 1694908798
    
    
    
    
    Done in 3.48s.

File: fees/hono.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      const graphRes: IData[] = (await request(graphUrls[chain], graphQuery, {
+        timestampFrom: fromTimestamp,
+        timestampTo: toTimestamp
+      })).dailyRevenueAggregators;
+
+      const prices = await getPrices(["ethereum:0x0000000000000000000000000000000000000000"], timestamp);
+      try {
+        const dailyRevenue = (Number(graphRes[0].todayETHRevenue))/10**18 * prices["ethereum:0x0000000000000000000000000000000000000000"].price;
+      return {
+        dailyRevenue: `${dailyRevenue}`,
```
Comment (dtmkeng) [inline fees/hono.ts:43]: how many fees ?

File: fees/hono.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        timestamp
+      }
+      } catch (error) {
+    
+      }
+      return {
```
Comment (dtmkeng) [inline fees/hono.ts:49]: may be this is should inside `catch` ?

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: fees/hono.ts (hunk 2 above)
Comment (dtmkeng) [reply fees/hono.ts:49]: how may bad catch should throw error like this `throw error`

File: fees/hono.ts (hunk 1 above)
Comment (nuoanunu) [reply fees/hono.ts:43]: Hi hi,  our protocol does not apply any fee to our end user. Our revenue came from our ETH-LINK LP, which we manage and share the profit among HONO holders.

File: fees/hono.ts (hunk 2 above)
Comment (nuoanunu) [reply fees/hono.ts:49]: There is only 1 case: where no revenue is found on that day. As we operate like a hedge fund, not everyday there will be revenue. 
*For example when the market is bad, no trading volumne => almost 0 to no revenue => Meaning then the revenue collector wont be active => no events => no data

**Comment by dtmkeng** [comment]
could you drop web site x account and more detail for list

**Comment by nuoanunu** [comment]
hi @dtmkeng here you go:
https://app.mangafi.org/
https://mangafi.org/

**Comment by dtmkeng** [comment]
is there tvl adapter already ?

**Comment by nuoanunu** [comment]
@dtmkeng yes I will create a PR for it later.

**Comment by nuoanunu** [comment]
hi hi @dtmkeng here is the TVL pr: https://github.com/DefiLlama/DefiLlama-Adapters/pull/7492#issuecomment-1722436099

**Comment by dtmkeng** [comment]
okay wait tvl list will list fees

**Pull request merged**